src/CARD_DATA.js

```diff
@@ -0,0 +1,68 @@
+export const CARD_DATA = [
+  {
+    name: "suzuki",
+    front: "images/Suzuki.png",
+    back: "images/Blue-BG.png",
+  },
+  {
+    name: "yamaha",
+    front: "images/Yamaha.png",
+    back: "images/Blue-BG.png",
+  },
+  {
+    name: "Mt10",
+    back: "images/Blue-BG.png",
+    front: "images/Mt10.png",
+  },
+  {
+    name: "jeep",
+    back: "images/Blue-BG.png",
+    front: "images/Jeep.png",
+  },
+  {
+    name: "mustang",
+    back: "images/Blue-BG.png",
+    front: "images/Mustang.png",
+  },
+  {
+    name: "350z",
+    back: "images/Blue-BG.png",
+    front: "images/350z.png",
+  },
+  {
+    name: "suzuki",
+    back: "images/Blue-BG.png",
+    front: "images/Suzuki.png",
+  },
+  {
+    name: "yamaha",
+    back: "images/Blue-BG.png",
+    front: "images/Yamaha.png",
+  },
+  {
+    name: "Mt10",
+    back: "images/Blue-BG.png",
+    front: "images/Mt10.png",
+  },
+  {
+    name: "jeep",
+    back: "images/Blue-BG.png",
+    front: "images/Jeep.png",
+  },
+  {
+    name: "mustang",
+    back: "images/Blue-BG.png",
+    front: "images/Mustang.png",
+  },
+  {
+    name: "350z",
+    back: "images/Blue-BG.png",
+    front: "images/350z.png",
+  },
+];
+
+export const CARDSTATE = {
+  FACEUP: "faceUp",
+  FACEDOWN: "faceDown",
+  MATCHED: "matched",
+};
```

Comment:
nice. this object makes sense to me. But this would be state for each card so I imagine this would be in the Card.js file